### PR TITLE
Add Pinned toot column

### PR DESCRIPTION
**Pull request by voidsatisfaction**
I just added Pinned toot section on web(react side)

It is easy for a mastodon user to access their own pinned toots.

related #4795

### CODE REVIEW

**Comment by voidsatisfaction** [comment]
## At Navigator, there is Pinned toots

screen shot 2017-09-06 at 6 32 10 pm

## Inside of Pinned toots

screen shot 2017-09-06 at 6 32 22 pm

File: app/controllers/api/v1/accounts/statuses_pins_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Api::V1::Accounts::StatusesPinsController < Api::BaseController
```
Comment (Gargron) [inline app/controllers/api/v1/accounts/statuses_pins_controller.rb]: There is already an API for getting an account's pinned toots... No need to add a new API.

Comment (voidsatisfaction) [reply app/controllers/api/v1/accounts/statuses_pins_controller.rb]: Thank you for reply! I will fix it.

File: app/javascript/mastodon/actions/pin_statuses.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+export const PINNED_STATUSES_FETCH_REQUEST = 'PINNED_STATUSES_FETCH_REQUEST';
+export const PINNED_STATUSES_FETCH_SUCCESS = 'PINNED_STATUSES_FETCH_SUCCESS';
+export const PINNED_STATUSES_FETCH_FAIL = 'PINNED_STATUSES_FETCH_FAIL';
+
+export function fetchPinnedStatuses() {
+  return (dispatch, getState) => {
+    dispatch(fetchPinnedStatusesRequest());
+
+    api(getState).get('/api/v1/accounts/1/statuses', { params: { pinned: true } }).then(response => {
```
Comment (abcang) [inline app/javascript/mastodon/actions/pin_statuses.js]: `'/api/v1/accounts/1/statuses'`

I think that it is necessary to access with the logged in user.

**Review by abcang — CHANGES_REQUESTED** [review]

File: app/javascript/mastodon/actions/pin_statuses.js (hunk 2 above)
Comment (voidsatisfaction) [reply app/javascript/mastodon/actions/pin_statuses.js]: Thanks! I totally forgot that

**Review by abcang — APPROVED** [review]

**Pull request merged**